channels/offline-channel-js/src/InMemoryBatch.ts

```diff
@@ -0,0 +1,122 @@
+import dynamicProto from "@microsoft/dynamicproto-js";
```

Comment:
nit: each file should have the copyright header (we probably don't have this on every file -- but we should)